src/catalog/distribute_iterator.cc

```diff
@@ -302,19 +302,22 @@ void DistributeWindowIterator::Next() {
         std::string cur_pk = kv_it_->GetPK();
         auto traverse_it = std::dynamic_pointer_cast<openmldb::base::TraverseKvIterator>(kv_it_);
         uint64_t last_ts = 0;
+        uint32_t ts_pos = 1;
```

Comment:
should be 0 or 1 if `traverse_it` is not valid?

Reply:
should be 0 if traverse from another tablet